### ISSUE TITLE
import from re-exported sometimes not working

### ISSUE THREAD

**Issue by anion155** [wessberg/rollup-plugin-ts#53]
Code:
```typescript
// items/Item.ts
export interface Item {
  id: string;
}
// items/index.ts
export * from './Item';
// someModule.ts
import { Item } from './items';
export class B {
  items: Item[];
}
```
Will lead to this result:
```typescript
interface Item {
    id: string;
}
declare class B {
  items: Item_$0[];
}
```
If I change import to `import { Item } from './items/Item'` everything working as expected.

**Comment by wessberg** [comment]
Hey there. Thanks for reporting this issue. It has been fixed on the [experiment](https://github.com/wessberg/rollup-plugin-ts/tree/experiment) branch and will be released with the next minor version.

**Comment by wessberg** [comment]
This has been fixed as part of [v1.2.0](https://github.com/wessberg/rollup-plugin-ts/releases/tag/v1.2.0). Happy New Year!
